operations/iuf/workflows/product_delivery.md

```diff
@@ -56,11 +56,26 @@ Refer to that table and any corresponding product documents before continuing to
 
 1. Invoke `iuf run` with activity identifier `${ACTIVITY_NAME}` and use `-r` to execute the [`deliver-product`](../stages/deliver_product.md) stage. Perform the upgrade using product content found in `${MEDIA_DIR}`.
 
-    (`ncn-m001#`) Execute the `deliver-product` stage.
+    1. Option 1: Install or upgrade without SLURM or PBS Products
 
-    ```bash
-    iuf -a ${ACTIVITY_NAME} -m "${MEDIA_DIR}" run -r deliver-product
-    ```
+       (`ncn-m001#`) Execute the `deliver-product` stage.
+
+       ```bash
+       iuf -a ${ACTIVITY_NAME} -m "${MEDIA_DIR}" run -r deliver-product
+       ```
+
+    1. Option 2: Install or upgrade along with SLURM or PBS Products
+
+       **`NOTE`** This subsection is mandatory only while installing SLURM or PBS Products. Additional arguments are available to control the behavior of the `deliver-product` stage, for example `-rv`. See the [`deliver-product`
+       stage documentation](../stages/deliver_product.md) for details and adjust the examples below if necessary.
+
+       1. Use site variables from the `site_vars.yaml` file found in `${ADMIN_DIR}` and recipe variables from the `product_vars.yaml` file found in `${ADMIN_DIR}`.
+
+           (`ncn-m001#`) Execute the `deliver-product` stage.
+
+           ```bash
+           iuf -a ${ACTIVITY_NAME} -m "${MEDIA_DIR}" run --site-vars "${ADMIN_DIR}/site_vars.yaml" -bpcd "${ADMIN_DIR}" -r deliver-product
+           ```
```

Comment:
I rewrote this section to add more details.

Reply:
In order to enable a workload manager, admin also need to include the site variables file with the `deploy-product` stage. I think we need to include these changes in the [Deploy product workflow](https://github.com/Cray-HPE/docs-csm/blob/release/1.6/operations/iuf/workflows/deploy_product.md) as well.

We should add similar options to step 2 of the [Execute the IUF `deploy-product` stage](https://github.com/Cray-HPE/docs-csm/blob/release/1.6/operations/iuf/workflows/deploy_product.md#1-execute-the-iuf-deploy-product-stage). Note that the `--site-vars` argument needs to be added, but the WLM does not need to be selected in the site variable file again (since it was already selected here for the `deliver-product` stage).